clang/lib/CIR/Dialect/Transforms/LoweringPrepare.cpp

```diff
@@ -162,6 +162,10 @@ void LoweringPreparePass::buildCXXGlobalInitFunc() {
   mlir::SymbolTable::setSymbolVisibility(
       f, mlir::SymbolTable::Visibility::Private);
   mlir::NamedAttrList attrs;
+  // 65535 is the default priority, i.e, the lowest priority.
+  // TODO: handle globals with a user-specified initialzation priority.
+  auto ctorAttr = mlir::cir::GlobalCtorAttr::get(builder.getContext(), 65535);
```

Comment:
Nice, few things here:

- For default values (like 65535), it'd be better if we add a custom builder to GlobalCtorAttr that uses a CArg with default value (see `GlobalOp` for examples).
- Spelling -> `initialization priority`
- Can you add an assert for user-specified initialization priority instead?